frontend/packages/schema-editor/src/components/SchemaInspector/ItemDataComponent.tsx

```diff
@@ -210,7 +211,7 @@ export function ItemDataComponent({ schemaNode }: IItemDataComponentProps) {
             aria-label={t('schema_editor.title')}
             onBlur={onChangeTitle}
             onChange={(e: ChangeEvent) => setItemItemTitle((e.target as HTMLInputElement)?.value)}
-            value={itemTitle}
+            value={itemTitle ?? ''}
```

Comment:
Is it necessary to fallback to an empty string? itemTitle is an empty string by default? 🤔 
```suggestion
            value={itemTitle}
```

Reply:
ah, we need the fallback here because the useEffect sets title and description and then override the default value to undefined.